pkgs/misc/vim-plugins/default.nix

```diff
@@ -953,6 +953,10 @@ rec {
       llvmPackages_38.llvm
     ] ++ stdenv.lib.optional stdenv.isDarwin Cocoa;
 
+    propogatedBuildInputs = [
```

Comment:
You also need to edit this file: https://github.com/bkase/nixpkgs/blob/016e8f26fea004c87c4a90372173c02046c90fe8/pkgs/misc/vim-plugins/vim2nix/additional-nix-code/youcompleteme